develop/tutorials/articles/xx-item-selector/creating-custom-entites-for-the-item-selector.markdown

```diff
@@ -71,13 +79,17 @@ Follow the steps below to create your custom Item Selector criterion:
         
         <!-- Is the @Activate @Override code needed here? I added it in because 
         I've seen it in other criterion handlers. For example: https://github.com/liferay/liferay-portal/blob/a5a0a3e961c2d87d565ca55d4df03985de72a4f6/modules/apps/collaboration/wiki/wiki-api/src/main/java/com/liferay/wiki/internal/item/selector/handler/WikiAttachmentItemSelectorCriterionHandler.java
+
+        Yes this is needed here in order to activate the component, since the
+        "default logic" is in the superclass and this method is called by OSGi
+        in th component activation
         -->
   
 This ties this handler class to the new criterion class you just created, and 
 activates the OSGi bundle, so it's available for use.
-<!-- Is the information above correct? -->
+<!-- Is the information above correct? YES-->
 
-The selection view is determined by the criterion specified in the 
+The selection view list is determined by the criterion specified in the
 `*ItemSelectorCriterionHandler` class. You can learn more about how selection 
```

Comment:
@robertoDiaz What is meant by "selection view list" ? How is that different from a selection view?